modules/tinymce/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Direct invalid child text nodes of list elements will be wrapped in list item elements. #TINY-4818
 
 ### Changed
+- The `link` plugin will no longer try to add, change, or remove links generated in the `tableofcontent` plugin. #TINY-9491
```

Comment:
Not sure how we should describe this but the change is now more general. The fix is going in to the core and `tableofcontent` is now a commercial plugin so I don't think we should be referencing this directly.

What about:
```suggestion
- The `link` plugins context menu items will no longer appear for noneditable links. #TINY-9491
```